web/src/components/snapshots/ConfigureSnapshots.jsx

```diff
@@ -26,6 +27,38 @@ class ConfigureSnapshots extends React.Component {
     });
   };
 
+  isVelero10OrNewer = () => {
+    const { snapshotSettings } = this.props;
+    if (!semverjs.valid(snapshotSettings?.veleroVersion)) {
+      return true;
+    }
+
+    const velero10Semver = semverjs.coerce("1.10");
+    const actualVeleroSemver = semverjs.coerce(snapshotSettings?.veleroVersion);
+
+    return semverjs.gte(actualVeleroSemver, velero10Semver);
+  };
```

Comment:
```suggestion
  isVelero10OrNewer = (
    valeroVersion = this.props.snapshotSettings?.veleroVersion) => {
    if (!semverjs.valid(veleroVersion)) {
      return true;
    }

    const velero10Semver = semverjs.coerce("1.10");
    const actualVeleroSemver = semverjs.coerce(veleroVersion);

    return semverjs.gte(actualVeleroSemver, velero10Semver);
  };
```
can remove the temporary variable by using default value

Reply:
yeah but that's not as readable IMO